tests/integration/test_actions.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration test for actions."""
+
+import pytest
+from juju.application import Application
+
+from .conftest import TEST_EXTERNAL_HOSTNAME_CONFIG
+
+
+@pytest.mark.abort_on_fail
+async def test_deploy(
```

Comment:
I feel like this should be named `test_get_certificate_action` or something similar rather than `test_deploy`. What do your think? Since this is such a small change it might be better to address in a separate PR (if you agree) to avoid blocking this from landing.